clients/js/package.json

```diff
@@ -48,6 +48,7 @@
     "@ava/typescript": "^4.1.0",
     "@solana-program/system": "^0.6.1",
     "@solana/eslint-config-solana": "^3.0.3",
+    "@solana/sysvars": "^2.0.0",
```

Comment:
This needs to be in either ` peerDependencies` or `dependencies`. Otherwise, the consumer of this package will be missing this dependency.